src/core/concepts/getting-started/diagrams.md

```diff
@@ -10,13 +10,13 @@ between them. The `draw` method plots the graph, with many other methods (
 [Matlab](https://github.com/datajoint/datajoint-matlab/blob/master/%2Bdj/ERD.m)) to
 save or adjust the output.
 
-Because DataJoint pipelines are directional (see [DAG](../../../glossary#dag)), the tables at the
+Because DataJoint pipelines are directional (see [DAG](../../../glossary#dag)), the 
+tables at the
```

Comment:
Was this intentional? Seems a bit funky to add a newline here and make this line short.

Reply:
Yes, I've been intentionally adding single line breaks to comply with the [markdown linter](https://github.com/DavidAnson/markdownlint). The motivation is that hard wrapping makes it easier to see small changes in future PRs. Should we change the [linter settings](https://github.com/datajoint-company/datajoint-docs/blob/main/.markdownlint.yaml)?

Reply:
Notes: @guzman-raphael and I discussed the shortcomings of this markdown linter. For PRs, it's useful to see separate editing and linting doesn't fit with the standard practice of a deterministic linter that runs for every commit.

@guzman-raphael - I've added another commit to fix the wrapping for this pgf. Please reapprove so we can merge here.